app/containers/Bookings/Details/QuickLook/index.js

```diff
@@ -604,6 +604,28 @@ OffenderManagementInCustody.defaultProps = {
   canUserViewProbationDocuments: false,
 }
 
+export const Pathfinder = ({ pathfinderUrl, isPathfinderUser }) => {
+  return (
+    <div>
+      {isPathfinderUser && pathfinderUrl && (
```

Comment:
think this should be moved to caller - otherwise if `pathfinderUrl` not defined there will be an empty div instead.

Reply:
Similar thing - the values passed into the Pathfinder function are then used in the unit-tests in pathfinder.test.js so it can vary the values supplied and conditionally render the link. You're right though - it is two checks, but the inner one enables the unit testing of this feature.